luminance/src/framebuffer.rs

```diff
@@ -145,16 +148,20 @@ where
       let repr = B::validate_framebuffer(repr)?;
 
       Ok(Framebuffer {
-        repr,
+        repr: ReprDropper(repr),
         color_slot,
         depth_slot,
       })
     }
   }
 
+  pub(crate) fn repr(&self) -> &B::FramebufferRepr {
```

Comment:
Why do you think adding this function is needed?

Reply:
I didn't want to expose the `ReprDropper` to the rest of the crate, it feels like a gory implementation hack that you don't want to have to think about when you're working on a different part of the codebase. Plus, I have an irrational fear of the tuple member access syntax so I wanted to hide that away in a single place too.

Reply:
Agreed, good argument. :)